lib/vagrant/util/guest_hosts.rb

```diff
@@ -2,11 +2,10 @@ module Vagrant
   module Util
     # Helper methods for modfiying guests /etc/hosts file
     module GuestHosts
-      # Linux specific inspection helpers
-      module Linux
 
+      module Unix
```

Comment:
maybe this naming can be better?

Reply:
We can definitely see if any other opinions show up. I'm leaning towards just having the modules named after the host (Linux, Darwin, Solaris, etc.) so we can provide overrides that are host specific where required and make usage of the modules a little more natural.